spec/shadow/index.html

```diff
@@ -1123,13 +1123,46 @@
       </section>
 
       <section>
-        <h3>Focus Navigation</h3>
+        <h3>Sequential Focus Navigation</h3>
 
         <p>If a <a>node</a> doesn’t <a data-lt="participates">participate</a> in the <a>document flat tree</a>, the <a>node</a> <strong>must</strong> be skipped from the <a>sequential focus navigation</a>.</p>
 
-        <p>The <a>sequential focus navigation order</a> for a given <a>shadow tree</a> <var>A</var> <strong>must</strong> be inserted into the <a>sequential focus navigation order</a> for the <a>parent tree</a> <var>B</var> as follows:</p>
+        <p>Each <a>shadow root</a> and <a>slot</a> element is a <a>control group owner</a> and
```

Comment:
My understanding is that frames own its own control group (e.g. `<iframe>`) and its own sequence,  but from user's standpoint, tab navigation sequence look *global*.  Thus sequences for each group is merged.